.github/workflows/testing.yml

```diff
@@ -48,12 +47,7 @@
 
        - name: install pulp via pip
          shell: bash -l {0}
-         run: pip install pulp 
-       
-       - name: install bleeding edge libpysal (only Ubuntu w/ latest Python)
-         shell: bash -l {0}
-         run: pip install git+https://github.com/pysal/libpysal.git@master
-         if: matrix.os == 'ubuntu-latest' && contains(matrix.environment-file, 'DEV')
+         run: pip install pulp
```

Comment:
Why is this here and not in the env yaml files?

Reply:
Previously there was problem with `pulp` being installed correctly from `conda`, even when done within the `pip` block. It had to done with only some of the available solvers being initialized. This would be a good time to revisit the process. Let me check that out.

Reply:
NOTE TO SELF -- I still need to look into this.